src/bundle/Resources/public/js/scripts/user_password.change.js

```diff
@@ -113,5 +114,6 @@
     oldPasswordInput.addEventListener('blur', (event) => checkIsNotEmpty(event.currentTarget), false);
     newPasswordInput.addEventListener('blur', (event) => checkIsNotEmpty(event.currentTarget), false);
     confirmPasswordInput.addEventListener('blur', (event) => checkIsNotEmpty(event.currentTarget), false);
+    newPasswordInput.addEventListener('blur', comparePasswords, false);
```

Comment:
Previously, if I changed the first password to match the second one it did not close the error notice.